appveyor.yml

```diff
@@ -40,23 +40,6 @@ environment:
       VCVARSALL: "x86"
       TEST1_CMD: ""
       TEST2_CMD: "py.test --maxfail 5 --durations=10 tests\\functional\\test_libraries.py"
-      
-    - PYTHON: "C:\\Python35-x64"
-      PYTHON_VERSION: "3.5"
-      PYTHON_ARCH: "64"
-      VCVARSALL: "x64"
-      # Break the tests into two runs, since together they exceed the 1 hour limit.
-      # See https://github.com/pyinstaller/pyinstaller/issues/2024#issuecomment-224129520
-      # for more discussion.
-      TEST1_CMD: "py.test --maxfail 5 --durations=10 tests\\unit tests\\functional -k \"not tests/functional/test_libraries.py\""
-      TEST2_CMD: ""
-
-    - PYTHON: "C:\\Python35-x64"
-      PYTHON_VERSION: "3.5"
-      PYTHON_ARCH: "64"
-      VCVARSALL: "x64"
-      TEST1_CMD: ""
-      TEST2_CMD: "py.test --maxfail 5 --durations=10 tests\\functional\\test_libraries.py"
```

Comment:
I take this as: We don't need to test the whole test-quite for both 64 and 32 bit. So for 64 bit we only run basic tests do make sure the bootloader is working correctly.